pom.xml

```diff
@@ -222,6 +222,11 @@
 			<version>3.5.4</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
```

Comment:
From Maven 3.9.0 and onwards, plexus-utils is no longer placed to plugin classpath by default (see [release notes](https://maven.apache.org/docs/3.9.0/release-notes.html)), so I had to add this one to ensure the code can run also when using recent Maven versions, which is the case in our environment.

3.2.1 is the one which works with `maven-plugins-api` of version 3.6.2 used by this plugin. See [dependency](https://maven.apache.org/ref/3.6.2/maven-plugin-api/dependencies.html) page.